File: Begineer/chocolateFight.c
#include<stdio.h>

int main(void)
{
  int result[]={}, count=0, testCase, mel, ban, i ;

  printf("Enter The Number Of Test Cases : ");
  scanf("%d",&testCase);

  for(int i=0;i<testCase;i++)
  {
    printf("Enter the numbers of chocolate : ");
    scanf("%d%d", &mel, &ban);
    chocolateLeft(mel, ban, count, result);
    count++;
  }

  for(i=0;i<=count;i++)
  {
    printf(result[i]);
  }
}

void chocolateLeft(int mel, int ban, int count, float result[])
{
  int leftChocolate;
  while(mel != ban || mel <= 0 || ban <= 0)
  {
    if(mel > ban)
    {
      mel = mel - ban;
    }
    else if(mel < ban)
    {
      ban = ban - mel;
    }
  }

  leftChocolate = mel + ban;
  result[count] = leftChocolate;
}

File: Begineer/chocolateFight.py
#PROBLEM https://www.codechef.com/problems/SINS

testCase = int(input("Enter the number of test case : "))
result = []

def chocolateLeft(mel, ban):
    while(mel != ban or mel <= 0 or ban <= 0):
        if(mel > ban):
            mel = mel - ban
        elif(mel < ban):
            ban = ban - mel
        else:
            pass

    leftChocolate = mel + ban
    result.append(leftChocolate)


for i in range(testCase):
    mel, ban = input().split()
    mel = int(mel)
    ban = int(ban)
    chocolateLeft(mel, ban)

for i in result:
    print(i)
